# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/vnc/vnc-xfree86.diff
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./xc/programs/Xserver/vnc/XserverDesktop.cc.orig	2004-09-13 06:57:27.998812712 +0200
+++ ./xc/programs/Xserver/vnc/XserverDesktop.cc	2004-09-13 06:57:46.886941280 +0200
@@ -382,25 +382,6 @@
   }
 }
 
-static void printRegion(RegionPtr reg)
-{
-  int nrects = REGION_NUM_RECTS(reg);
-
-  fprintf(stderr,"Region num rects %2d extents %3d,%3d %3dx%3d\n",nrects,
-          (REGION_EXTENTS(pScreen,reg))->x1,
-          (REGION_EXTENTS(pScreen,reg))->y1,
-          (REGION_EXTENTS(pScreen,reg))->x2-(REGION_EXTENTS(pScreen,reg))->x1,
-          (REGION_EXTENTS(pScreen,reg))->y2-(REGION_EXTENTS(pScreen,reg))->y1);
-
-  for (int i = 0; i < nrects; i++) {
-    fprintf(stderr,"    rect %3d,%3d %3dx%3d\n",
-            REGION_RECTS(reg)[i].x1,
-            REGION_RECTS(reg)[i].y1,
-            REGION_RECTS(reg)[i].x2-REGION_RECTS(reg)[i].x1,
-            REGION_RECTS(reg)[i].y2-REGION_RECTS(reg)[i].y1);
-  }
-}
-
 CARD32 XserverDesktop::deferredUpdateTimerCallback(OsTimerPtr timer,
                                                    CARD32 now, pointer arg)
 {
--- ./xc/programs/Xserver/Imakefile.orig	2004-09-13 07:22:05.729163552 +0200
+++ ./xc/programs/Xserver/Imakefile	2004-09-13 07:22:25.665132824 +0200
@@ -118,7 +118,7 @@
        LIBREGEX = RegexLibrary
 
 #if DoLoadableServer
-    LIBCWRAPPER = os/libcwrapper.o
+    LIBCWRAPPER = os/libcwrapper.o os/libos.a
 #endif
 
 #if BuildXprint
